bazel/envoy_mobile_dependencies.bzl

```diff
@@ -42,6 +42,8 @@ def kotlin_dependencies():
             "androidx.test:monitor:1.3.0",
             "androidx.test.ext:junit:1.1.2",
             "org.robolectric:robolectric:4.4",
+            "org.hamcrest:hamcrest:2.2",
```

Comment:
Would it be alright if we stick with `"org.assertj:assertj-core:3.12.0",` instead of adding hamcrest?

Reply:
I couldn't find the corresponding `Matchers` class/methods in assertj. 
See usage [here](https://github.com/colibie/envoy-mobile/blob/ee7e49eeb196fb28b58c6ca1ae2eff61c5c542d6/test/java/org/chromium/net/urlconnection/CronetFixedModeOutputStreamTest.java#L473)

Reply:
Hamcrest does not seem to be imported anywhere - is it really needed?

Reply:
https://github.com/colibie/envoy-mobile/blob/ee7e49eeb196fb28b58c6ca1ae2eff61c5c542d6/test/java/org/chromium/net/urlconnection/CronetFixedModeOutputStreamTest.java

It's imported in this test file.

Reply:
Given the fact that we try to keep the ported tests intact, and it would not be easy to change, I guess this is fine fine.

Reply:
Ah. Seems like this isn't that big of a priority to avoid.